### DISCUSSION TITLE
lab4問題IP config 設置問題以及資料輸入

### DISCUSSION THREAD

**Discussion by lumusen0305** [bol-edu/HLS-SOC-Discussions#213] [Q&A]
Hello，大家好。目前做Lab4遇到兩個問題想請教一下
問題一、
我按照Lab1 Fir的地址去設定tap 跟 len(64筆data) (地址都跟lab1的offset一樣，但是結果會報出Warning)。
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/45762207/a7cad5a4-6c43-4fbb-bba0-1f985f036578)
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/45762207/62d07b27-52f9-4cbe-92f3-d419a6eb31b4)
問題二、
輸入的值我目前是設置輸入0-63不知道這樣設置o不ok因為edge_detect的輸入挺大的這樣改不知道會部會出事
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/45762207/a39c3094-4081-4a59-9064-0abd86a013f8)

**Comment by ruei7916** [comment]
問題一可能與 #214 有關

**Reply by lumusen0305** [reply]
了解，感覺可以嘗試把fir塞到prj0，等下來試試看。感謝
